toc.js

```diff
@@ -138,4 +138,4 @@ window.addEventListener( 'scroll', function () {
     }
 }, false );
 
-}( document, window ) );
+}( document, window ) );
```

Comment:
Please remove this unnecessary change.

Reply:
I have try to remove that line but I realize that line is related to the first ligne in the toc.js file